### ISSUE TITLE
Add reporting of the daily growth rate

### ISSUE THREAD

**Issue by seabbs** [epiforecasts/epinowcast#176]
As of #152, the model uses a growth rate or reproduction number-based generative process and stores this as a summary value. However, when a generation interval is used this summary statistic changes the definition (i.e from a growth rate to a reproduction number). In most circumstances, this is likely to be unexpected. 

We should instead estimate the growth rate using post-processing and consistently return it - this would also make downstream post-processing (i.e in #175) easier. 

See here for an extended discussion of this: https://github.com/epinowcast/epinowcast/pull/152#discussion_r995976516

**Comment by adrian-lison** [comment]
Yeah, I would vote for treating everything inside the stan model as reproduction number and calling it capital R, and to add the growth rate via post-processing. We could even have a latent and a reference growth rate...

**Comment by seabbs** [comment]
Great idea to report both kinds of growth rate. Will need to document that clearly but its often not clear what people actually want so making a choice/only giving the latent is definitely not the best answer.
